src/hostedPages/activities/form/form.module.scss

```
@@ -52,3 +52,11 @@
   align-items: center;
   width: 100%;
 }
+
+.traditional_form {
+  margin-bottom: 0px;
+}
+
+.traditional_form_question {
+  margin-bottom: var(--awell-spacing-6);
```

Just tested a couple of values and think 8 works best.

```suggestion
  margin-bottom: var(--awell-spacing-8 );
```